config/prod/tom-cvat-alb.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a load balancer for minidream.synapse.org
+template_path: remote/managed-alb-https.yaml
+stack_name: tom-cvat-alb
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
+  VpcId: "vpc-0e9b80dc470a797d5"   # computevpc
+  AppPort: "80"         # rstudio port
```

Comment:
your app is running on port 80? also comment is "rstudio"?

Reply:
Theres two ports. 80 and 8082.

Reply:
hmm, this might be a problem.  will need to take a look at how we might do this.